flink-tests/src/test/java/org/apache/flink/test/checkpointing/NotifyCheckpointAbortedITCase.java

```diff
@@ -97,8 +99,10 @@
  */
 @RunWith(Parameterized.class)
 public class NotifyCheckpointAbortedITCase extends TestLogger {
+	private static final Logger LOG = LoggerFactory.getLogger(NotifyCheckpointAbortedITCase.class);
```

Comment:
No need to add this logger. You can use the `log` instance from `TestLogger`

Reply:
Resolved in latest commit.